### ISSUE TITLE
Model training problems

### ISSUE THREAD

**Issue by yangmingweiwtu** [georgeNakayama/AIpparel-Code#20]
Hi authors, I trained the model according to your readme.md. The difference between my model and yours is the number of GPUs. I used two H100s. However, the model loss did not decrease during training. I tried modifying the lr, batch size, and optimizer, but to no work. Could you please provide some suggestions?

<img width="2056" height="880" alt="Image" src="https://github.com/user-attachments/assets/ed2a52f4-1492-4ecc-b3d0-9512affdb9f6" />

**Comment by yangmingweiwtu** [comment]
The last question is, is there a starting weight for all parameter fine-tuning?

**Comment by yangmingweiwtu** [comment]
> Hi authors, I trained the model according to your readme.md. The difference between my model and yours is the number of GPUs. I used two H100s. However, the model loss did not decrease during training. I tried modifying the lr, batch size, and optimizer, but to no work. Could you please provide some suggestions?
> 
> <img alt="Image" width="2000" height="880" src="https://private-user-images.githubusercontent.com/201517323/489393289-ed2a52f4-1492-4ecc-b3d0-9512affdb9f6.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTgwOTMwMjUsIm5iZiI6MTc1ODA5MjcyNSwicGF0aCI6Ii8yMDE1MTczMjMvNDg5MzkzMjg5LWVkMmE1MmY0LTE0OTItNGVjYy1iM2QwLTk1MTJhZmZkYjlmNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwOTE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDkxN1QwNzA1MjVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00ZDViYmE1OWUyYjllY2NmMWNlYWNmMjllNDRkNjVjMjE4ZDYyOTIyYmExYTllYzliZTQ0MGYzZmMwMWFkYTllJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ML731bs7X-RgD-yLC-6R4y1f9Xk5Csn09vj26LZmE6A">
Resolved！

**Comment by yangmingweiwtu** [comment]
> The last question is, is there a starting weight for all parameter fine-tuning?

Questions remaining.

**Comment by georgeNakayama** [comment]
Hi, 

For starting weight, do you mean the weight we initialized for training?

**Comment by yangmingweiwtu** [comment]
> Hi,
> 
> For starting weight, do you mean the weight we initialized for training?

Yes, that’s what I meant.

**Comment by yangmingweiwtu** [comment]
> Hi,
> 
> For starting weight, do you mean the weight we initialized for training?

Yes, that's what I meant. Is it fine-tuning all parameters on the weights of Llava-7b?

**Comment by georgeNakayama** [comment]
We used LLAVA 1.5 7b to initialize our training. It is given in this line: https://github.com/georgeNakayama/AIpparel-Code/blob/master/configs/aipparel.yaml#L19 .

**Comment by yangmingweiwtu** [comment]
> We used LLAVA 1.5 7b to initialize our training. It is given in this line: https://github.com/georgeNakayama/AIpparel-Code/blob/master/configs/aipparel.yaml#L19 .

Thank you for your reply! There is no mention of downloading the weight file of llava-7b in the readme.md. Is it possible that the program will download it automatically?

**Comment by georgeNakayama** [comment]
Hi, it should download automatically since it will pull from Huggingface based on the tag.

**Comment by yangmingweiwtu** [comment]
> Hi, it should download automatically since it will pull from Huggingface based on the tag.

OK, I understand. Thank you！
